lib/generators/surveyor/templates/db/migrate/add_is_key_column_to_questions.rb

```diff
@@ -0,0 +1,12 @@
+# encoding: UTF-8
+# frozen_string_literal: true
+
+class AddIsKeyColumnToQuestions < ActiveRecord::Migration[5.2]
+  def self.up
+    add_column :questions, :is_key, :boolean, default: false, null: false
+  end
+
+  def self.down
+    remove_column :questions, :is_key
+  end
+end
```

Comment:
```suggestion
end

```
I think your IDE should take care of this.

Reply:
I'll fix this in my follow on for version bumps.

Reply:
I'll fix in my follow on for version bumps